### ISSUE TITLE
Modifier Or Pairs not work properly

### ISSUE THREAD

**Issue by WesRateur** [balderdashy/sails-redis#52]
I have this condition :
```javascript
Model.find({
    or: [{
        id: 25
    }, {
        id: 26
    }]
})
```
If I do this it work and two entries are retrieved.
However, if the two id isn't consecutive like this : 
```javascript
Model.find({
    or: [{
        id: 25
    }, {
        id: 27
    }]
})
```

Only one entry will be retrieve.
PS: do you know a better choice to retrieve 2 entries with ids?

**Comment by Ryanc1256** [comment]
@WesRateur you can do 
Model.find({
     id: [25, 26]
})

I think, i haven't tested it, but it should work, since it's the same in most of the other adapters.

**Comment by sailsbot** [comment]
Thanks for posting, @WesRateur.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
